### DISCUSSION TITLE
Question about camera plugin

### DISCUSSION THREAD

**Discussion by joeamansfield** [runelite/runelite#16185] [Q&A] [answered]
I ran across the setting "Reset zoom position" in the camera plugin, but I'm confused about what that is actually supposed to do. Looking up the camera plugin gave me this explanation: "The position to which zoom is reset if the Reset zoom option of Control Function is selected. (Default 512, Min -400, Max 1004)," but I was unable to figure out what Control Function is supposed to be.
Any explanation would be awesome, thanks!

**Comment by Nightfirecat** [comment, marked as answer]
The "Control Function" config has 3 options:

1. `None`, which means it does not do anything special with the camera
2. `Hold to zoom`, which means using the scroll wheel to zoom is disabled unless the Ctrl key is held down
3. `Reset zoom`, which sets the camera zoom level to the configured amount when Ctrl is pressed
